clang/tools/clang-offload-extract/ClangOffloadExtract.cpp

```diff
@@ -0,0 +1,179 @@
+//===-------- clang-offload-extract/ClangOffloadExtract.cpp ---------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+///
+/// \file
+/// Implementation of the clang-offload-extract tool which allows extracting
+/// target images from linked fat offload binaries. For locating target images
+/// in the binary it uses information from the .tgtimg section which is added to
+/// the image by the clang-offload-wrapper tool. This section contains <address,
+/// size> pairs for all embedded target images.
+///
+//===----------------------------------------------------------------------===//
+
+#include "clang/Basic/Version.h"
+#include "llvm/ADT/ArrayRef.h"
+#include "llvm/Object/COFF.h"
+#include "llvm/Object/ELFObjectFile.h"
+#include "llvm/Object/ObjectFile.h"
+#include "llvm/Support/CommandLine.h"
+#include "llvm/Support/Errc.h"
+#include "llvm/Support/Signals.h"
+#include "llvm/Support/WithColor.h"
+#include "llvm/Support/raw_ostream.h"
+
+#define IMGAGE_INFO_SECTION_NAME ".tgtimg"
```

Comment:
```suggestion
#define IMAGE_INFO_SECTION_NAME ".tgtimg"
```

Reply:
Thanks! I did not notice that I have made a typo here.